application/frontend/package.json

```diff
@@ -28,7 +28,8 @@
     "react-error-boundary": "^4.0.13",
     "react-helmet-async": "^2.0.5",
     "react-hook-form": "7.49.3",
-    "react-icons": "5.0.1"
+    "react-icons": "5.0.1",
+    "sharp": "^0.33.5"
```

Comment:
Is this used anywhere? Please remove if unused to reduce overhead.

Reply:
yes they are the icons on the floating boxes